docs(website): improve formatting, remove an erroneous link (#67)
biomejs/biome: website/src/pages/formatter/index.mdx

# Formatter

> Biome's an opinionated formatter that has the goal to stop all ongoing debates over styles.

Reviewer: ```suggestion
Biome is an opinionated formatter that has the goal to stop all ongoing debates over styles.
```

> It follows a similar [philosophy to Prettier](https://prettier.io/docs/en/option-philosophy.html),
> only supporting few options to avoid that debates over styles turn into debates over Biome options.
> It deliberately [resists the urge to add new options](https://github.com/prettier/prettier/issues/40) to prevent [bike-shed discussions](https://en.wikipedia.org/wiki/Law_of_triviality) in teams so they can focus on what really matters instead.

Reviewer: ```suggestion
It follows a similar [philosophy to Prettier](https://prettier.io/docs/en/option-philosophy.html),
only supporting a few options to avoid debates over styles, turning into debates over Biome options.
It deliberately [resists the urge to add new options](https://github.com/prettier/prettier/issues/40) to prevent [bike-shed discussions](https://en.wikipedia.org/wiki/Law_of_triviality) in teams so they can focus on what really matters instead.
```

## Ignoring Code

> There are times when the way we format code might not be ideal.
> For these cases you can use a format suppression comment:

Reviewer: ```suggestion
There are times when the formatted code isn't ideal.

For these cases, you can use a format suppression comment:
```
